src/components/Icon/icons.js

```diff
@@ -1,229 +1,41 @@
 export const map = {
```

Comment:
Are we positive this is the full set of icons? It would be nice to test this against Janus to make sure we aren't removing icons are that being used...